lib/ansible/modules/cloud/amazon/aws_batch_job_queue.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Jon Meran <jonathan.meran@sonos.com>
```

Comment:
Needs to follow copyright guidelines http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright